data/schema.sql

```diff
@@ -0,0 +1,34 @@
+
+CREATE DATABASE IF NOT EXISTS `voice_transcriptions` DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci;
+
+# Table to hold transcriptions, where each word is a record
+CREATE TABLE `transcriptions` (
+    `id` int NOT NULL AUTO_INCREMENT,
+    `conversation_uuid` varchar(255) NOT NULL,
+    `channel` varchar(255) NOT NULL,
+    `start_time` varchar(10) NOT NULL,
```

Comment:
Question really, but is there a reason to define it as a `varchar` here and not `datetime` or something equivalent?

Reply:
Good question. The start_time in this case is not a typical timestamp format. Instead it is like '000.00' where the 3 digits on the left of the decimal are seconds, and the right are microseconds.